### ISSUE TITLE
Adaptive Asymmetry

### ISSUE THREAD

**Issue by JesseL10** [Alibaba-MIIL/ASL#93]
Hi! First of all thank you for this interesting work!

I'm trying to apply this loss function to my multi-label dataset and to choose a good set of hyper-parameters. I'm wondering how the code dynamically change the hyper-parameter gamma_neg mentioned in Section 2.7.  According to Eq.(11), there's a parameter named lambda, I'm not sure what value should I set to this one. Could you please share the setup of this part? 

 In addition, my dataset is a highly imbalanced one, and there are a lot of samples with labels which are all negative, which means during each iteration, there may appear a situation where all labels are all 0s. Do you think this technique would work on my dataset?

Thank you a lot!

**Comment by nj888-art** [comment]
Hello, I'm encountering a similar problem to yours. 

Could you please share the solution you used? 

I look forward to your reply.
